apps/prairielearn/src/pages/home/home.sql

```diff
@@ -1,5 +1,20 @@
 -- BLOCK select_home
 WITH
+  admin_institutions AS (
+    -- Note that we only consider institutions where the user is explicitly
+    -- added as an administrator. We do not include all institutions if the
+    -- user is a global administrator, as that would be a very long list.
+    --
+    -- Global admins can access institutions/courses via the admin pages.
```

Comment:
I fully agree that we only want to show an institution link on the homepage if the user is explicitly an institution admin for it.

However, I couldn't actually figure out how to access the institution admin page hierarchy as a global admin, except by using the link on the global-admin's "Admins" tab to go to to the institution-admin's "Admin" tab. Is there supposed to be a way to get to the institution admin pages from the global admin institution pages (or from somewhere else for global admins)?

One suggestion here is to unify the two hierarchies, but have global admins see some extra tabs. That is, we'd move all `institutionAdmin*` and `administratorInstitution*` pages to a single set of `institution*` pages. Both the homepage links (for institution admins) and the links from the global admin "Institutions" page would go to these same pages. However, the nav would only show a subset of these pages unless you are a global admin, and the corresponding individual pages would require global admin to access.

Reply:
Yeah, the only way for a global admin to reach this page at the moment is via the "admins" tab in the administrator institution pages. As discussed elsewhere, we're going to maintain the two separate hierarchies for now so that we don't have to think too carefully about permissions on the global admin pages. We can revisit this if we want to increase the functionality of the institution admin pages.

Reply:
Yep, agreed.